frontend/src/App.tsx

```diff
@@ -999,6 +999,7 @@ export class App extends PureComponent<Props, State> {
       pageName = document.location.pathname
         .replace(`/${basePath}`, "")
         .replace(new RegExp("^/?"), "")
+        .replace(new RegExp("/$"), "")
```

Comment:
the reason why I like Python is that in python you can use str.removesuffix('/') instead of using regex-es :)